doc/release-notes-24464.md

```diff
@@ -0,0 +1,8 @@
+Notable changes
+===============
+
+Tools and Utilities
+-------------------
+
+- Allow logging with severity level and display it with the category name. (#24464)
```

Comment:
Not sure, but a release note doesn't seem needed until severity level is a user-facing choice.

Reply:
Agreed, removed this commit 👍🏻